CMakeCompiler.txt

```diff
@@ -31,9 +31,14 @@ if(WIN32 AND NOT DEFINED RSTUDIO_PACKAGE_BUILD)
   set(RTOOLS_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../dependencies/windows/Rtools33")
   string(REGEX REPLACE "/" "\\\\" RTOOLS_TOOLCHAIN_32BIT_PATH "${RTOOLS_PATH}/mingw_32/bin")
   set(ENV{PATH} "${RTOOLS_TOOLCHAIN_32BIT_PATH}")
+  set(WINDRES "${RTOOLS_TOOLCHAIN_32BIT_PATH}\\windres.exe")
```

Comment:
For some reason, on my Windows machine I was seeing that CMake 3.8 couldn't find `windres.exe` when executed at e.g.

https://github.com/rstudio/rstudio/blob/f92d5a37a69ab5d51376e9cef35c6d528cf49519/src/cpp/desktop/CMakeLists.txt#L251-L268

I'm not yet sure if that was something locally weird in my environment, or if `add_custom_command()` no longer searches for commands on the PATH (or somehow that wasn't propagated to that CMake call).